### PR TITLE
CE-3694: Display proper colors on dark wikis

### PR DESCRIPTION
**Pull request by lukaszkonieczny**

### CODE REVIEW

File: extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss [hunk 1]
```diff
@@ -84,6 +85,8 @@ $close-button-background: '/resources/wikia/ui_components/modal/images/close-dar
 	}
 
 	.pmp-entry-point-close {
+		background: url('/resources/wikia/ui_components/modal/images/close-dark.svg');
+		background-size: 15px 15px;
```
Comment (kamilkoterba) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss]: You can use just background-image

File: extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss [hunk 2]
```diff
@@ -75,6 +75,7 @@ $close-button-background: '/resources/wikia/ui_components/modal/images/close-dar
 	z-index: 2;
 
 	p.pmp-entry-point-message {
+		color: #333;
```
Comment (kamilkoterba) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss:79]: What about using $color-text-dark  from skins/shared/color.scss ?

Comment (lukaszkonieczny) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss:79]: `$color-text-dark` is different color ;)

Comment (kamilkoterba) [inline extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss:79]: And probably the right one to use :)

**Comment by kamilkoterba** [comment]
LGTM

**Pull request merged**